SConstruct

```diff
@@ -1466,18 +1471,6 @@ else :
 	if not haveInkscape :
 		sys.stderr.write( "WARNING : Inkscape not found - not building docs. Check INKSCAPE build variable.\n" )
 
-#########################################################################################################
-# Documentation reference scripts
-#########################################################################################################
-
-referenceFiles = []
-for ext in ( 'grf', 'png' ) :
-	referenceFiles += glob.glob( "doc/references/*.%s" % ext )
-
-for f in referenceFiles :
-	fileInstall = env.Command( f.replace( "doc/", "$BUILD_DIR/resources/", 1 ), f, Copy( "$TARGET", "$SOURCE" ) )
-	env.Alias( "build", fileInstall )
-
```

Comment:
Can we just update the preceding commit so we add the above in the first time, rather than adding this, and removing it here?

Reply:
Sorry about that, sorted in 209795c1499d4448301e7f0d99453d9c48b1b798.